content/_data/authors/YimingGong.adoc

```diff
@@ -3,4 +3,5 @@ name: "Yiming Gong"
 github: "Cr1t-GYM"
 ---
 
-GSoC 2022 Contributor.
+Yiming is an Electrical and Computer Engineering student at Carnegie Mellon University.
+He is the Jenkins GSoC contributor in 2022 who developed link:https://jenkinsci.github.io/jfr-action-doc[a near Feature-complete version of Jenkinsfile Runner Actions as GitHub Actions].
```

Comment:
This sentence sounds a bit awkward. Probably change it to something like "He developed ... during GSoC '22".